platform/source/lib/ax/platform/container_specs.py

```diff
@@ -187,9 +188,13 @@ def __init__(self, size_in_mb):
         libmodule_hostpath.set_type("HOSTPATH", "/lib/modules")
         self.add_volume(libmodule_hostpath)
 
+        cluster_name_id = os.getenv("AX_CLUSTER_NAME_ID", None)
+        assert cluster_name_id, "Cluster name id is None!"
+        cluster_config = AXClusterConfig(cluster_name_id=cluster_name_id)
```

Comment:
You should be able to instantiate AXClusterConfig without an kwargs and it should still work.